curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8f1d5f5ddbf74c07f733b.md

```diff
@@ -11,12 +11,19 @@ In the `lose` function, call the `update` function and pass in the sixth object
 
 # --hints--
 
-Test 1
+You should call the `update` function inside the `lose` function.
 
 ```js
+assert.match(lose.toString(), /update/)
```

Comment:
So technically this test only checks if the camper wrote `update` inside the `lose` function. 
It doesn't test if the camper calls it though.
We should update this test to check if the `update` function called

This should work

```suggestion
assert.match(lose.toString(), /update\(/);
```